### DISCUSSION TITLE
PX are the only units used for custom breakpoints?

### DISCUSSION THREAD

**Discussion by do-kevin** [tailwindlabs/tailwindcss#10264] [Help] [answered]
Hey there folks,

So I'm using a mixture Tailwind's default responsive breakpoints and arbitrary values. Howver, I wanted to use `em` for my media queries because `px` appears to be inconsistent with a few other browsers especially Safari. The breakpoints are not working as of now as I am trying to fix the issues on Safari.

I've tried something like `min-[46.875em]:` for example, but the breakpoint doesn't appear to trigger. Is this the right Tailwind behavior for configuring custom breakpoints? It only works with `px`, not the other units like `em`, `rem`, `ch`, etc...? `px` media queries have appeared to work on chromium and firefox browsers so far in my testing.

Ref:
[https://tailwindcss.com/docs/responsive-design#using-custom-breakpoints](url)

**Comment by do-kevin** [comment]
> Hey there folks,
> 
> So I'm using a mixture Tailwind's default responsive breakpoints and arbitrary values. Howver, I wanted to use `em` for my media queries because `px` appears to be inconsistent with a few other browsers especially Safari. The breakpoints are not working as of now as I am trying to fix the issues on Safari.
> 
> I've tried something like `min-[46.875em]:` for example, but the breakpoint doesn't appear to trigger. Is this the right Tailwind behavior for configuring custom breakpoints? It only works with `px`, not the other units like `em`, `rem`, `ch`, etc...? `px` media queries have appeared to work on chromium and firefox browsers so far in my testing.
> 
> Ref: [https://tailwindcss.com/docs/responsive-design#using-custom-breakpoints](url)

Nevermind, seems like my issues with Safari was due to not setting min-heights on my elements, not media-queries.

**Comment by adamwathan** [comment, marked as answer]
Hey! You can use any units you want, the only thing that matters is that you only use _one_ unit and don't mix and match. So if you want to use `em`, make sure you configure your default screens in your config to use `em`, then your arbitrary `min-*` and `max-*` screens will also work with `em`.

Here's a working demo:

https://play.tailwindcss.com/wnaMdSwHvk?size=652x720

**Reply by do-kevin** [reply]
Splendid, thanks dude!
